nncf/experimental/torch/quantization/quantize_model.py

```diff
@@ -87,7 +87,7 @@ def send_to_device(tensor):
 def quantize_impl(
     model: torch.nn.Module,
     calibration_dataset: Dataset,
-    preset: QuantizationPreset,
+    preset: Optional[QuantizationPreset],
```

Comment:
Shouldn't it be like this?

```suggestion
    preset: Optional[QuantizationPreset] = None,
```

Otherwise it would fail if `preset` parameter is not given, despite it being marked as optional.

Also similar cases below.

Reply:
it does not apply to this function. Optional[QuantizationPreset] is the same Union[None, QuantizationPreset]. I can replace if you have concern about Optional.